### PR TITLE
add background-color support

### PR DESCRIPTION
**Pull request by jqnatividad**
Implements #4

### CODE REVIEW

File: assets/javascripts/bbcode_color_dialect.js [hunk 1]
```diff
@@ -6,6 +6,15 @@
...
+    while (text != (text = text.replace(/\[bgcolor=([^\]]+)\]((?:(?!\[bgcolor=[^\]]+\]|\[\/bgcolor\])[\S\s])*)\[\/bgcolor\]/ig, function (match, p1, p2, offset, string) {
+      return "<span style='background-color: " + p1 + "'>" + p2 + "</span>";
+    })));
+    return text;
+  }
+
   Discourse.Dialect.addPreProcessor(replaceFontColor);
+  Discourse.Dialect.addPreProcessor(replaceFontBgColor);
   Discourse.Markdown.whiteListTag('font', 'color');
+  Discourse.Markdown.whiteListTag('span', 'style');
```
Comment (ZogStriP) [inline assets/javascripts/bbcode_color_dialect.js]: Hmm `style` is kind of a *dangerous* attribute to whitelist...
I would only support whitelisting changes of the `background-color` value.
Can you fix that?

**Comment by jqnatividad** [comment]
@ZogStriP done! Hope it gets accepted so I can update several docker-hosted instances :)

**Comment by jqnatividad** [comment]
And you can check it in action here 
https://talk.beta.nyc/t/highlighting-text/223

**Comment by ZogStriP** [comment]
Awesome thanks! Could you rebase and squash into 1 commit?

**Comment by jqnatividad** [comment]
rebased and squashed :)

**Pull request merged**